src/picongpu/include/particles/gasProfiles/FromHDF5Impl.hpp

```diff
@@ -74,7 +75,12 @@ struct FromHDF5Impl : public T_ParamClass
     {
         using namespace splash;
         DataConnector &dc = Environment<>::get().DataConnector();
-        FieldTmp& fieldTmp = dc.getData<FieldTmp > (FieldTmp::getName(), true);
+
+        PMACC_CASSERT_MSG(
```

Comment:
every time you want to run `dc.getData` you assume the slot you access is really allocated in the current simulation. the assert checks that the user did not fool you on that and that `getData()` will not throw at RT.